docs/eventing/sugar/README.md

```diff
@@ -38,16 +35,33 @@ the default settings:
 
 There might be cases where automated Broker creation is desirable, such as on
 namespace creation, or on Trigger creation. The Sugar controller enables those
-use-cases:
+use-cases. The following sample configuration of the `sugar-config` ConfigMap
```

Comment:
Do we really think this should be the default to be documented? If it is then mention something about the option to use a label to restrict where the behavior is applied by using the global configmap `config-sugar`

Reply:
Thanks for the feedback, added a label selector for the namespaces which would limit the application to be more intentional. The Trigger example shows how to unleash it everywhere.

Reply:
Also fixed some other inconsistencies that I had missed earlier.